docs/content/preview/architecture/docdb-sharding/colocated-tables.md

```diff
@@ -65,20 +65,22 @@ This is characteristic of a microservices-oriented architecture, where each micr
 
 **Example:** Multi-tenant SaaS services where one database is created per customer. As new customers are rapidly on-boarded, it becomes necessary to add more databases quickly while maintaining high-availability and fault-tolerance of each database.
```

Comment:
Under "Motivation" at the top of the page:

This feature is desirable in a number of scenarios, some of which are described below. --> Should be rephrased in general and "below" shouldn't be used.